src/index.ts

```diff
@@ -423,12 +514,65 @@ export class FlashbotsBundleProvider extends providers.JsonRpcProvider {
     })
   }
 
+  private waitForTxInclusion(transactionHash: string, startBlockNumber: number, timeout: number, maxBlockNumber?: number) {
```

Comment:
This function is specifically for privateTx, and should probably be named accordingly - eg. `waitForPrivateTxInclusion`.

having 2 parameters startBlockNumber and maxBlockNumber is also confusing, I think it would be better to have only maxBlockNumber

Reply:
Good point, `startBlockNumber` was only being used to calculate `maxBlockNumber` if it wasn't specified. 
Removed `startBlockNumber` and made `maxBlockNumber` required.